src/main/java/com/omvoid/community/utils/JsonResultWriter.java

```diff
@@ -0,0 +1,47 @@
+package com.omvoid.community.utils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.omvoid.community.exception.JsonWriterException;
+import com.omvoid.community.models.CoreExpansionResults;
+import com.omvoid.community.models.DefaultCoreExpansionResults;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class JsonResultWriter implements ResultWriter {
+    @Override
+    public void writeResult(CoreExpansionResults<String> results, String path) throws JsonWriterException {
+        writeAsJson(results.getCommunitiesMap(), path + "\\CommunitiesMap.json");
+        writeAsJson(results.getVertexCoresMap(), path + "\\VertexCoresMap.json");
+        writeAsJson(results.getVertexCommunityMap(), path + "\\VertexCommunityMap.json");
```

Comment:
use `File.separator` instead of `\\`
let's move filenames to final static fields